circle.yml

```diff
@@ -0,0 +1,20 @@
+machine:
+  node:
+    version: 4.2.1
+  ruby:
+    version: 2.2.3
+
+dependencies:
+  pre:
+    - gem install github_changelog_generator
+    - pip install awscli
```

Comment:
Add ruby gem for making changelogs.  Add awscli for s3 sync.  These are both used in the deploy script.